internals/config/default.yml

```diff
@@ -0,0 +1,252 @@
+        alternate_text: "openwisp"
+      links:
+        - text:
+            en: "Sign In"
+          url: "/default/login"
+          authenticated: false
+        - text:
+            en: "Sign Up"
+          url: "/default/registration"
+          authenticated: false
+        - text:
+            en: "Change password"
+          url: "/default/change-password"
+          authenticated: true
+
+    footer:
+      links:
+        - text:
+            en: "link-1"
+          url: "https://link-1.com"
+          authenticated: false
+        - text:
+            en: "link-2"
+          url: "https://link-2.com"
+          authenticated: true
+          verified: false
+        - text:
+            en: "link-3"
+          url: "https://link-3.com"
+          authenticated: true
+          verified: true
+      secondary_text: false
+
+    registration_form:
+      input_fields:
+        phone_number:
+          type: "tel"
+        username:
+          type: "text"
+          pattern: '[a-zA-Z@.+\-_\d]{1,150}'
+        email:
+          type: "email"
+          pattern: '.+@.+\..+'
+        password:
+          type: "password"
+          pattern: ".{6,}"
+        password_confirm:
+          type: "password"
+          pattern: ".{6,}"
+        first_name:
+          type: "text"
+          setting: "disabled"
+        last_name:
+          type: "text"
+          setting: "disabled"
+        birth_date:
+          type: "date"
+          setting: "disabled"
+        location:
+          type: "text"
+          pattern: '[a-zA-Z@.+\-_\d]{1,150}'
+          setting: "disabled"
+        country:
+          type: "text"
+          pattern: '[a-zA-Z@.+\-_\d\s]{1,150}'
+        zipcode:
+          type: number
+        street:
+          type: "text"
+        city:
+          type: "text"
+        tax_number:
+          type: "text"
+          pattern: '[a-zA-Z@.+\-_\d]{1,150}'
+      additional_info_text: true
+      links:
+        login: true
+
+    password_reset_form:
+      additional_text: true
+      input_fields:
+        email:
+          type: "email"
+          pattern: '.+@.+\..+'
+      contact_text: true
+      login_page_link: true
+
+    password_reset_confirm_form:
+      additional_text: true
+      input_fields:
+        password:
+          type: "password"
+          pattern: ".{6,}"
+        password_confirm:
+          type: "password"
+          pattern: ".{6,}"
+      contact_text: true
+      login_page_link: true
+
+    contact_page:
+      email: "support@openwisp.co"
+      helpdesk: "+789 948 564"
+      social_links:
+        - alt:
+            en: "twitter"
+          icon: "twitter.svg"
+          url: "https://twitter.com/openwisp"
+        - alt:
+            en: "facebook"
+          icon: "facebook.svg"
+          url: "https://facebook.com/openwisp"
+
+    status_page:
+      links:
+        - text:
+            en: "Change your password"
+          url: "/{orgSlug}/change-password/"
```

Comment:
I noticed that here it's named `{orgSlug}`, but above `{org_slug}`, do you know why?

Reply:
Because it is replaced in the client code.